Changelog.cfg

```diff
@@ -0,0 +1,34 @@
+
+KERBALCHANGELOG //Required to have this name
+{
+	showChangelog = True //To show the changelog, this must be set to True
+	modName = KDex //Add your mod's name here
+	VERSION //Declares a version node
+	{
+		version = 1.11.3.11 //Version number, numbers only with no spaces!
+		change = zer0Kerbal played with one of his favorite mods
+		change = added masTerTourch/Localization/
+		change = added localization (en-us.cfg)
+		change = adjusted phrasing a smudge
+		change = - electostatic -> electrostatic
+		change = - incomming -> incoming
+		change = - phosporic -> phosphoric
+		change = - munar -> Munar
```

Comment:
....
change = Added localisation and fixed some typos.
change = Updated model textures and tweaked part configuration.
change = Modernised mod layout and added additional metadata (Changelog)

(Less is more?)

Reply:
in change logs, especially when in situations like this - I often over-communicate - the changelog was for you - am from a pre-git universe. 📟 so often best way was document changes verbatim. 

If too much, no worries. truncate as you see fit (obviously).

Reply:
Ah I see, and much appreciated.  I too hail from pre-git, but nowadays find modern diff tools to give me all the details I need with a human changelog reduced to pointing out any subtle changes and providing high-level summaries.  With git, best practice is to have each (set of) changes as a single commit. 

In practice I still tend to code away, and then try to unpick logically-cohesive commits afterwards - again, luckily this is one of git's strong points especially with a powerful gui like Git Extensions.